### ISSUE TITLE
Syntax Highlighting not working (50 % of the time)

### ISSUE THREAD

**Issue by john-cj** [asciidoctor/asciidoctor-browser-extension#300]
(I'm aware about issue https://github.com/asciidoctor/asciidoctor-browser-extension/issues/255 with a very similar title, but the issue itself is different.)

Test file:

```
:source-highlighter: highlightjs

[source, html]
----
<div class="foo">
----

[source, html]
----
<div class="foo">
----

[source]
----
<div class="foo">
----

[source, html]
----
<div class="foo">
----
```

Refresh the browser page multiple times. The refresh rate doesn't matter.

In Chrome:

* 50 % of the time the highlighting will be disabled.
* 50 % of the time the highlighting will work for 1st, 2nd, and 3rd blocks.

**Comment by Mogztter** [comment]
Hello @john-cj and thanks for your report.
I can reproduce this issue. It looks like a "timing" issue where depending on which actions get executed first, the highlighting will work or not.

I will need to get a closer look at code to understand what is going on.

**Comment by Mogztter** [comment]
Please note that the third block will not be highlighted because the language is not defined:

```adoc
[source]
----
<div class="foo">
----
```

The other blocks should be highlighted but for some reasons the following script is not always executed:

```js
if (window.hljs) {
  var blocks = document.querySelectorAll('pre code')
  blocks.forEach(window.hljs.highlightBlock)
}
```

We might need to highlight the code before updating the content on the page (ie. server-side highlighting).

**Comment by john-cj** [comment]
> Please note that the third block will not be highlighted because the language is not defined:

I cannot reproduce the bug right now, but it seems it was a typo: I mean 1st, 2nd, and 4th blocks. 

(I was in a hurry, also the internet access was nearly broken for 2 days.)

**Comment by Mogztter** [comment]
I've just released and published version 2.3.0: https://github.com/asciidoctor/asciidoctor-browser-extension/releases/tag/v2.3.0
Could you please confirm that this issue is now resolved?
